README.md

```diff
@@ -74,7 +74,29 @@ $version = XdebugHandler::getSkippedVersion();
 ```
 
 ### Output
-The library is silent unless the `-vvv` command-line option is present, which results in the output of simple status messages.
+The `setLogger` method enables the output of status messages to an external PSR3 logger.
+
+```php
+use Composer\XdebugHandler\XdebugHandler;
+
+$xdebug = new XdebugHandler('myapp');
+// Provide a PSR3 logger
+$xdebug->setLogger($myLogger);
+```
+
+All messages are prefixed `xdebug-handler:` and are reported with either `DEBUG` or `WARNING` log levels. No context data is provided. For example:
```

Comment:
I think if you are using a library like monolog you'll likely want to have the logs from this go to an XDebugHandler _channel_ which would handle prefixing the log message in a consistent way with other channels (and allows to send different channels to different destinations etc). so I think prefixing every log message is harmful here because it would duplicate that.
If you  are not using monolog it's easy enough for a consumer to prefix the messages in the Logger implementation.

https://github.com/Seldaek/monolog/blob/master/doc/01-usage.md#leveraging-channels

Reply:
And it is also easy enough for a consumer to strip the prefix  :) Monolog channels are great, but surely they shouldn't concern us here.

Reply:
Is prefixing common in other tools?

Reply:
I’m wondering the same

Reply:
One solution would be to stretch the SRP principle slightly by including a cli logger as a tiny convenience class (there is already a logger in the tests anyway, so no bloat) that would accept an existing output stream (or default to stderr) and prefix each message?

Reply:
No, please leave the cli logger out 😄